content/guides/integrations/deepchem.md

```diff
@@ -29,70 +29,85 @@ If you'd rather dive straight into working code, check out this [**Google Colab*
 
 Setup Weights & Biases for DeepChem models of type [KerasModel](https://deepchem.readthedocs.io/en/latest/api_reference/models.html#keras-models) or [TorchModel](https://deepchem.readthedocs.io/en/latest/api_reference/models.html#pytorch-models).
 
-1. Install the `wandb` library and log in
+### Sign up and create an API key
 
-    {{< tabpane text=true >}}
+An API key authenticates your machine to W&B. You can generate an API key from your user profile.
 
-    {{% tab header="Command Line" value="cli" %}}
+{{% alert %}}
+For a more streamlined approach, you can generate an API key by going directly to [https://wandb.ai/authorize](https://wandb.ai/authorize). Copy the displayed API key and save it in a secure location such as a password manager.
+{{% /alert %}}
 
-    ```
-    pip install wandb
-    wandb login
-    ```
+1. Click your user profile icon in the upper right corner.
+1. Select **User Settings**, then scroll to the **API Keys** section.
+1. Click **Reveal**. Copy the displayed API key. To hide the API key, reload the page.
+
+### Install the `wandb` library and log in
 
-    {{% /tab %}}
+To install the `wandb` library locally and log in:
 
-    {{% tab header="Notebook" value="notebook" %}}
+{{< tabpane text=true >}}
+{{% tab header="Command Line" value="cli" %}}
 
-    ```python
-    !pip install wandb
+1. Set the `WANDB_API_KEY` [environment variable]({{< relref "/guides/models/track/environment-variables.md" >}}) to your API key.
 
-    import wandb
-    wandb.login()
+    ```bash
+    export WANDB_API_KEY=<your_api_key>
     ```
 
-    {{% /tab %}}
+1. Install the `wandb` library and log in.
 
-    {{< /tabpane >}}
 
-2. Initialize and configure WandbLogger
 
-    ```python
```

Comment:
`WandbLogger` refers to a Class that makes it easier to log checkpoints, metrics, etc. for folks using DeepChem.

Reply:
I.e. So you still need the import statements.